tests/sequence/test_sequence.h

```diff
@@ -1061,7 +1061,14 @@ void testSequenceBack(TString & /*Tag*/)
 
     // val is a reference in contrast to the const version of back().
     TReference val = back(string);
+#if defined(__GNUC__) && (__GNUC__ == 11) // Spurious error for GCC11
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstringop-overflow"
+        val = 'A';
```

Comment:
I tried around for a bit, couldn't replicate the warning. This seems to be bogus.

Reply:
whitespaces? 8 instead of 4?